libcst/_nodes/tests/test_atom.py

```diff
@@ -1120,6 +1121,8 @@ def test_invalid(self, **kwargs: Any) -> None:
         )
     )
     def test_versions(self, **kwargs: Any) -> None:
+        if is_native() and not kwargs.get("expect_success", True):
+            self.skipTest("parse errors are disabled for native parser")
```

Comment:
Do you have a plan for making them work?  This is an important part of the API.

Reply:
Yeah it's possible to make it work, but my priority is to get 3.10 working first. The old parser won't be removed until this feature is supported by the new one.